### PR TITLE
DEV-19117 Upgrade .NET from 6.0.21 to 6.0.22

### PR DESCRIPTION
**Pull request by mtesseracttech**

### CODE REVIEW

File: FontoXPathCSharp/FontoXPathCSharp.csproj [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
     <PropertyGroup>
         <TargetFramework>net6.0</TargetFramework>
+        <RuntimeFrameworkVersion>6.0.22</RuntimeFrameworkVersion>
+		<Version>7.0.0</Version>
```
Comment (JosVerburg) [inline FontoXPathCSharp/FontoXPathCSharp.csproj:6]: Please remove the `<Version>7.0.0</Version>` it is not needed for the dotnet upgrade and we don't need to specify version yet in the stage we're at imo

File: PrscSharp/PrscSharp.csproj [hunk 2]
```diff
@@ -2,6 +2,8 @@
 
     <PropertyGroup>
         <TargetFramework>net6.0</TargetFramework>
+        <RuntimeFrameworkVersion>6.0.22</RuntimeFrameworkVersion>
+		<Version>7.0.0</Version>
```
Comment (JosVerburg) [inline PrscSharp/PrscSharp.csproj:6]: See previous comment

File: XPathBenchmarks/XPathBenchmarks.csproj [hunk 3]
```diff
@@ -3,6 +3,8 @@
     <PropertyGroup>
         <OutputType>Exe</OutputType>
         <TargetFramework>net6.0</TargetFramework>
+        <RuntimeFrameworkVersion>6.0.22</RuntimeFrameworkVersion>
+		<Version>7.0.0</Version>
```
Comment (JosVerburg) [inline XPathBenchmarks/XPathBenchmarks.csproj:7]: See previous comment

File: XPathRunner/XPathRunner.csproj [hunk 4]
```diff
@@ -3,6 +3,8 @@
     <PropertyGroup>
         <OutputType>Exe</OutputType>
         <TargetFramework>net6.0</TargetFramework>
+        <RuntimeFrameworkVersion>6.0.22</RuntimeFrameworkVersion>
+		<Version>7.0.0</Version>
```
Comment (JosVerburg) [inline XPathRunner/XPathRunner.csproj:7]: See previous comment

File: XPathTest/XPathTest.csproj [hunk 5]
```diff
@@ -2,8 +2,9 @@
 
     <PropertyGroup>
         <TargetFramework>net6.0</TargetFramework>
+        <RuntimeFrameworkVersion>6.0.22</RuntimeFrameworkVersion>
+		<Version>7.0.0</Version>
```
Comment (JosVerburg) [inline XPathTest/XPathTest.csproj:6]: See previous comment

**Review by JosVerburg — CHANGES_REQUESTED** [review]

**Review by Erikvl87 — COMMENTED** [review]
Requesting the same changes as @JosVerburg reported regarding not needing to include the `<Version>` elements.

**Review by JosVerburg — APPROVED** [review]

**Review by Erikvl87 — APPROVED** [review]

**Comment by JosVerburg** [comment]
It's merged.. but seems GitHub is not a fan of me merging locally instead of using the merge button ¯\_(ツ)_/¯

**Pull request closed without merging**